bot_modules/irc_commands.js

```diff
@@ -116,6 +116,32 @@ module.exports = {
 			bot.say(info.channel, `String "${title}" does not match entry %title%;`)
 		})
 	},
+	
+	/**
+	 *	entryid
+	 *
+	 */
+	entryid: (info, words) => {
+		let title = words.slice(1).join(' ')
```

Comment:
i would do this differently.

1) you should rename `title`. this argument is not the title. it's the id
2) you should not be joining all of the arguments
3) you should guard against people providing too many arguments and provide usage info like

```
invalid arguments. usage: /entryid <id>
```

Reply:
i would also actually guard against providing something that's doesn't look like an ID. you can probably use a regex:

```
const idPattern = /^\d+$/;
if (!idPattern.test(id)) {
  bot.say(info.channel, `"${id}" doesn't look like an id to me`);
}
```